biot_savart_integrators/generic/integrand.py

```diff
@@ -13,6 +14,16 @@
 def biot_savart_integrand(r, r_j, j, spatial_dim):
     R = r - r_j
     numerator = cross(j, R, spatial_dim)
-    denominator = (R**2).sum(dim=spatial_dim)**(3/2.)
+    denominator = (R**2).sum(dim=spatial_dim) ** (3 / 2.)
     integrand = _SI_FACTOR * numerator / denominator
-    return integrand
\ No newline at end of file
+    return integrand
+
+
+def biot_savart_potential_integrand(r: xr.DataArray, r_j: xr.DataArray,
```

Comment:
I'm open to a better naming convention of these functions. I'm not sure if the "B-S law" relates by default only to the mag field., so perhaps something like  `B_biot_savart_integrand` and `A_biot_savart_integrand` would be more readable? Not sure myself yet.

Reply:
I would use `biot_savart_integrand_A` and `biot_savart_integrand_B` not to vailotent PEP8 toooooo much.

Reply:
your choice looks ok, let's go with that